### ISSUE TITLE
bug(ui): Progressing Applications are narrower in Applications List

### ISSUE THREAD

**Issue by rbreeze** [argoproj/argo-cd#16288]
Happens in `main`:

**Comment by JessieTeng89** [comment]
Hi @rbreeze 
Can you assign this bug to me? I will create a PR for it.
Thanks!

**Comment by rbreeze** [comment]
Thanks @JessieTeng89 ! If you cannot reproduce this, feel free to close this issue. I spoke with some other folks who could not reproduce, and it's very possible this was happening due to my local config.

**Comment by JessieTeng89** [comment]
@rbreeze You are right, I tried in my local environment, I cannot reproduce this issue.

**Comment by andrii-korotkov-verkada** [comment]
ArgoCD versions 2.10 and below have reached EOL. Can you upgrade and let us know if the issue is still present, please?
